cmd/updateprs/updateprs.go

```diff
@@ -109,7 +115,7 @@ func runClose(c *cobra.Command, _ []string) {
 	// Prompting for confirmation
 	if !yesFlag {
 		// TODO: add the number of PRs that it will actually close
-		if !p.AskConfirm(fmt.Sprintf("Close %s campaign PRs for all repos in %s?", dir.Name, repoFile)) {
+		if !p.AskConfirm(fmt.Sprintf("Close %s campaign PRs for all repos in %s", dir.Name, repoFile)) {
```

Comment:
Didn't like the double question mark here
<img width="753" alt="image" src="https://github.com/user-attachments/assets/d095f5a3-f551-44fc-b4b0-a11047929da6" />